collective/elasticsearch/indexes.py

```diff
@@ -389,12 +400,10 @@ def get_value(self, object):
     def get_query(self, name, value):
         value = self._normalize_query(value)
         date = value.ISO8601()
-        return {
-            'and': [
-                {'range': {'%s.%s1' % (name, name): {'lte': date}}},
-                {'range': {'%s.%s2' % (name, name): {'gte': date}}}
-            ]
-        }
+        return [
+            {'range': {'{}.{}1'.format(name, name): {'lte': date}}},
```

Comment:
you can use `'{0}.{0}1'.format(name)` instead of passing the same argument twice; format is expensive.